etna/models/holt_winters.py

```diff
@@ -277,6 +277,126 @@ def get_model(self) -> HoltWintersResultsWrapper:
         """
         return self._result
 
+    def _check_mul_components(self):
+        """Raise error if model has multiplicative components."""
+        model = self._model
+
+        if model is None:
+            raise ValueError("This model is not fitted!")
+
+        if (model.trend is not None and model.trend == "mul") or (
+            model.seasonal is not None and model.seasonal == "mul"
+        ):
+            raise ValueError("Forecast decomposition is only supported for additive components!")
+
+    def _rescale_components(self, df: pd.DataFrame, components: pd.DataFrame) -> pd.DataFrame:
+        """Rescale components when Box-Cox transform used."""
```

Comment:
Rescale components after Box-Cox inverse transformation

Reply:
This caption is not valid because inverse transformations not applied before this step. In any case it may be misleading, because we do just rescaling only when `use_boxcox` is used.